recipe/meta.yaml

```diff
@@ -7,19 +7,13 @@ package:
 
 source:
   url: https://github.com/scipy/scipy/archive/v{{ version }}.tar.gz
-  sha256: 91269acb6d073f09743e8b6549fe5f8adb4329218a4e2c027074b54ba68c1e22
+  sha256: 3e3bdd41e726e35ef576e46374278e2193aa19fbf5905214355153e65c280e1f
   patches:
-    - gh10591.patch
-    - relax_gmres_error_check.patch  # [aarch64]
-    - skip_problematic_boost_test.patch  # [aarch64 or ppc64le]
-    - skip_problematic_root_finding.patch  # [aarch64 or ppc64le]
+    # - relax_gmres_error_check.patch  # [aarch64]
+    # - skip_problematic_boost_test.patch  # [aarch64 or ppc64le]
+    # - skip_problematic_root_finding.patch  # [aarch64 or ppc64le]
     - skip_TestIDCTIVFloat_aarch64.patch  # [aarch64]
     - skip_white_tophat03.patch  # [aarch64 or ppc64le]
```

Comment:
These 2 are still needed?

Reply:
I did not check and I do not know. Shall we test without?

Reply:
yes, let's check